docker/dev/docker-compose.yml

```diff
@@ -46,5 +54,117 @@ services:
+    build:
+      context: ../../rafiki
+      dockerfile: ./packages/auth/Dockerfile
+    restart: always
+    networks:
+      - testnet
+    ports:
+      - '3007:3006'
+    environment:
+      NODE_ENV: development
+      AUTH_DATABASE_URL: postgresql://auth:auth@postgres/auth
+      INTROSPECTION_HTTPSIG: "false"
+    depends_on:
+      - tigerbeetle
+      - postgres
+  backend:
+    image: ghcr.io/interledger/rafiki-backend:latest
+    build:
+      context: ../../rafiki
+      dockerfile: ./packages/backend/Dockerfile
+    restart: always
+    privileged: true
+    ports:
+      - '3010:80'
+      - '3011:3001'
+    networks:
+      - testnet
+    environment:
+      NODE_ENV: development
+      LOG_LEVEL: debug
+      ADMIN_PORT: 3001
+      CONNECTOR_PORT: 3002
+      OPEN_PAYMENTS_PORT: 80
+      DATABASE_URL: postgresql://backend:backend@postgres/backend
+      TIGERBEETLE_CLUSTER_ID: 0
+      # Tigerbeetle will support DNS in future
+      TIGERBEETLE_REPLICA_ADDRESSES: '["10.5.0.50:4343"]'
+      NONCE_REDIS_KEY: test
+      AUTH_SERVER_GRANT_URL: http://auth:3006
+      AUTH_SERVER_INTROSPECTION_URL: http://auth:3006/introspect
+      ILP_ADDRESS: test.rafiki
+      STREAM_SECRET: BjPXtnd00G2mRQwP/8ZpwyZASOch5sUXT5o0iR5b5wU=
+      ADMIN_KEY: admin
+      PUBLIC_HOST: http://backend
+      OPEN_PAYMENTS_URL: http://backend
+      REDIS_URL: redis://redis:6379/0
+      PAYMENT_POINTER_URL: https://backend/.well-known/pay
+      # testnet urls - not implemented
+      WEBHOOK_URL: http://testnet/webhooks
+      PRICES_URL: http://testnet/prices
+      QUOTE_URL: http://testnet/quotes
+    depends_on:
+      - tigerbeetle
+      - postgres
+      - redis
+  tigerbeetle:
+    image: ghcr.io/tigerbeetledb/tigerbeetle@sha256:621268959801084246679f25416a9451f553fe3697dd4f618f39503322079433
+    restart: unless-stopped
+    privileged: true
+    volumes:
+      - tigerbeetle-data:/var/lib/tigerbeetle
+    networks:
+      testnet:
+        ipv4_address: 10.4.0.50
+    entrypoint:
+      - /bin/sh
+      - -c
+      - |
+        set -ex
+        DATA_FILE=/var/lib/tigerbeetle/cluster_0_replica_0.tigerbeetle
+        set +e
+        ls $$DATA_FILE
+        DATA_FILE_EXISTS="$$?"
+        set -e
+        echo $$DATA_FILE_EXISTS
+        if [ "$$DATA_FILE_EXISTS" != 0 ]; then 
+          /opt/beta-beetle/tigerbeetle format --cluster=0 --replica=0 $$DATA_FILE; 
+        fi
+        hostname -I
+        ls /var/lib/tigerbeetle
+        /opt/beta-beetle/tigerbeetle start --addresses=0.0.0.0:4343 $$DATA_FILE
+  redis:
+    image: 'redis:7'
+    restart: unless-stopped
+    networks:
+      - testnet
+  signatures:
+    build:
+      context: ../../rafiki
+      dockerfile: ./packages/http-signature-utils/Dockerfile
+    restart: always
+    ports:
+      - '3042:3000'
+    environment:
+      KEY_FILE: /workspace/private-key.pem
+    volumes:
+      - ./private-key.pem:/workspace/private-key.pem
+    networks:
+      - testnet
```

Comment:
Do we need the signatures application? 🤔

https://github.com/interledger/rafiki/tree/main/packages/http-signature-utils#http-signature-utils-library

Reply:
looks like no, we will add it back if we will need it in the future